datasets/noaa_nclimgrid/tests/test_dataset.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from pathlib import Path
+
+from pctasks.cli.cli import setup_logging, setup_logging_for_module
+from pctasks.dev.test_utils import run_process_items_workflow
+
+HERE = Path(__file__).parent
+DATASET_PATH = HERE / ".." / "dataset.yaml"
+
+
+# Change the COG_CONTAINER in noaa_nclimgrid.py to "devstoreaccount1" to avoid
+# uploading COGs to the nclimgridwesteurope storage account
```

Comment:
It's likely out-of-scope for this PR, but this would be a nice-to-have as a configuration argument to `run_process_items_workflow`.

Reply:
Agreed. However, I don't see a straightforward path (as I peruse the code) for that.

In a related topic, I asked Rob about adding extra containers for COGs or GeoParquet files to the `dataset.yaml` files, but he preferred to keep those types of things out. I think, to him, that can get into messy scope creep where we start bolting on cruft to the config yaml. He was fine with a hardcoded path. I'm not a fan.